src/Adapter/Cache/MemcacheServerManager.php

```diff
@@ -78,23 +78,23 @@ public function addServer($serverIp, $serverPort, $serverWeight)
      * Test if a Memcache configuration is valid.
      *
      * @param string $serverIp
-     * @param string $serverPort
+     * @param int $serverPort
      *
      * @return bool
      */
     public function testConfiguration($serverIp, $serverPort)
     {
         if (extension_loaded('memcached')) {
             $memcached = new Memcached();
-            $memcached->addServer($serverIp, $serverPort);
+            $memcached->addServer($serverIp, (int) $serverPort);
```

Comment:
Is it really needed, the method signature already ask for an int :thinking:

Reply:
Yes, the behavior is the same as `public function addServer($serverIp, $serverPort, $serverWeight)`. As the the parameter is not typed.